spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -33,7 +37,9 @@ is non-portable and may vary between implementations. For portable behavior, mon
 
 === Scope
 
-Metrics added by this specification will appear in the `base` MicroProfile Metrics scope.
+In MicroProfile Metrics, metrics added by this specification will appear in the `base` MicroProfile Metrics scope.
+
+In MicroProfile Telemetry, the generation of metrics is bounded to individual applications.
```

Comment:
I'm not quite sure what this means or what it implies for how the user is going to see the metrics.

I can see you've tried to add an equivalent statement for Telemetry here, to match the one for Metrics. However, I don't think Telemetry has a concept of scopes so I don't think we need this sentence at all.